management-service/services/segmenter_service.go

```diff
@@ -249,10 +249,8 @@ func (svc *segmenterService) ListSegmenters(
 			}
 			// UpdatedAt and CreatedAt fields are manually updated for custom segmenters but not global segmenters since
 			// global segmenters do not contain these fields
-			customSegmenterUpdatedAt := customSegmenter.UpdatedAt
-			customSegmenterCreatedAt := customSegmenter.CreatedAt
-			formattedCustomSegmenter.UpdatedAt = &customSegmenterUpdatedAt
-			formattedCustomSegmenter.CreatedAt = &customSegmenterCreatedAt
+			formattedCustomSegmenter.UpdatedAt = &customSegmenters[idx].UpdatedAt
+			formattedCustomSegmenter.CreatedAt = &customSegmenters[idx].CreatedAt
```

Comment:
Slipped this in in response to https://github.com/gojek/turing-experiments/pull/19#discussion_r923221867.